packages/api-db-mongodb/src/db/memberPlan.ts

```diff
@@ -119,7 +120,7 @@ export class MongoDBMemberPlanAdapter implements DBMemberPlanAdapter {
 
     // TODO: Rename to search
     if (filter?.name !== undefined) {
-      textFilter.$and?.push({name: {$regex: filter.name, $options: 'i'}})
+      textFilter.$and?.push({name: {$regex: escapeRegExp(filter.name), $options: 'i'}})
```

Comment:
Nice work. Could you please add the `escapeRegExp()` method to the `textFilter` methods in the other database adapters.